qgepqwat2ili/__init__.py

```diff
@@ -130,18 +136,49 @@ def main(args):
         SCHEMA = config.ABWASSER_SCHEMA
         ILI_MODEL = config.ABWASSER_ILI_MODEL
         ILI_MODEL_NAME = config.ABWASSER_ILI_MODEL_NAME
+        
         if args.direction == "export":
             utils.ili2db.create_ili_schema(
                 SCHEMA, ILI_MODEL, make_log_path(log_path, "ilicreate"), recreate_schema=args.recreate_schema
             )
             qgep_export(selection=args.selection.split(",") if args.selection else None, labels_file=args.labels_file)
-            utils.ili2db.export_xtf_data(SCHEMA, ILI_MODEL_NAME, args.path, make_log_path(log_path, "iliexport"))
+            
+            # utils.ili2db.export_xtf_data(SCHEMA, ILI_MODEL_NAME, args.path, make_log_path(log_path, "iliexport"))
+            
+            # additional parameter export_model_name needed
+            # export SIA405_ABWASSER_2015_LV95
+            #if args.export_model_name == "SIA405_ABWASSER_2015_LV95":
+            #    ILI_EXPORT_MODEL_NAME = config.ABWASSER_ILI_EXPORT_MODEL_NAME
+            #12.1.2023
+            #if args.export_model_name == config.ABWASSER_ILI_EXPORT_MODEL_NAME:
+            if args.export_model_name:
+                print("--export_model_name selected!")
+                # 12.1.2023
+                ILI_EXPORT_MODEL_NAME = config.ABWASSER_ILI_EXPORT_MODEL_NAME
+                
+                utils.ili2db.export_xtf_data(SCHEMA, ILI_EXPORT_MODEL_NAME, ILI_EXPORT_MODEL_NAME, args.path, make_log_path(log_path, "iliexport"))
+            else:
+                # additional parameter export_model_name needed
+                # export VSA_KEK_2019_LV95 - ILI_EXPORT_MODEL_NAME stays empty ''
+                utils.ili2db.export_xtf_data(SCHEMA, ILI_MODEL_NAME, '', args.path, make_log_path(log_path, "iliexport"))
```

Comment:
I dont see how the ILI_EXPORT_MODEL_NAME is used (see below about the export_xtf_data function)